public/Lab-Files/Lesson02/Challenges/lists.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+</head>
+<body>
+    <h3>Places I want to visit</h3>
```

Comment:
I would suggest using an h1 since you only used one heading. You can change the font size with CSS.